File: 기초 사고력문제/백준_2775_부녀회장이 될테야/부녀회장이 될테야.java
import java.util.*;

public class Main{
    public static void main(String[] args){
        Scanner s = new Scanner(System.in);
        int[][] apt = new int[15][15];

        for(int i = 0; i < apt.length; i++){
            for(int j = 1; j< apt[i].length; j++ ){
                if(i == 0) apt[i][j] = j;
                else{
                    if(j == 1) apt[i][j] = j;
                    else apt[i][j] = apt[i-1][j] + apt[i][j-1];
                }
            }
        }
        int t = s.nextInt();
        int k, n;
        while(t-- > 0){
            k = s.nextInt();
            n = s.nextInt();
            System.out.println(apt[k][n]);
        }


    }
}

File: 기초 사고력문제/백준_2775_부녀회장이 될테야/부녀회장이 될테야.py
apt = [[0 for _ in range(15)] for _ in range(15)]

for i in range(len(apt)):
    for j in range(len(apt[i])):
        if i == 0: apt[i][j] = j
        else:
            if j == 1: apt[i][j] = 1
            elif j > 1: apt[i][j] = apt[i-1][j] + apt[i][j-1]

t = int(input())
while t > 0 :
    t-=1
    k = int(input())
    n = int(input())
    print(apt[k][n])